frontend/src/pages/Seller/Dashboard/SellerOrders.tsx

```diff
@@ -2,12 +2,22 @@ import * as React from "react";
 import { Order } from "../../../components/shared/types/Order";
 import { PLACEHOLDER_ORDERS } from "../../../constants/placeholder-data";
 import DisplayedCustomerOrderItem from "../../Buyer/DisplayedCustomerOrderItem";
+import {useAPIHook} from "../../../components/shared/hooks/use-api-hook.ts";
 
-const SellerOrders: React.FC<{}> = (props) => {
+const SellerOrders: React.FC = () => {
     const [orders, setOrders] = React.useState<Order[]>(PLACEHOLDER_ORDERS);
-
-    // TODO: Fetch orders from our API
-
+    const apiHook = useAPIHook();
+    React.useEffect(() => {
+        const fetchOrders = async () => {
+            try {
+                const response = await apiHook.get(`http://localhost:3000/api/orders?itemOwnerId=65f8eb04ccc06864028d51f6`);
+                setOrders(response.data.orders);
```

Comment:
These are the placeholder orders:

![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/b1c44936-16b0-43fc-8b16-41d83e097d88)

Use `response.orders` instead here. That should fix the issue